lang/ilias_hu.lang

```diff
@@ -9331,7 +9331,7 @@ chatroom_adm#:#https#:#HTTPS
 chatroom_adm#:#chat_enabled#:#Csevegés engedélyezése
 chatroom_adm#:#enable_osd#:#Tartalomtár-csevegés: Képernyő-értesítések engedélyezése
 chatroom_adm#:#general_settings_title#:#Általános csevegésbeállítások
-chatroom_adm#:#osd_intervall#:#Frissítési intervallum
+chatroom_adm#:#osd_interval#:#Frissítési intervallum
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.